### ISSUE TITLE
Android emulator for Windows

### ISSUE THREAD

**Issue by iampossessed** [iampossessed/globalaltruism#5]
I need a good Android emulator for Windows. Will it be great to code it myself? How much of experience will I need to have? I just need to play Android games on PC, not a big deal though.

**Comment by SabinaWellington** [comment]
I am not sure that you need to code this emulator by yourself. From what I heard, companies hire teams of developers to do so. Considering the fact that you need to use it for personal matters, I find independent coding excessive. Go find a ready-made solution.

**Comment by iampossessed** [comment]
Right. Initially, all inside me rebelled against this suggestion, as my personal importance feeling was hurt. But after some thinking, I agreed to invest this time into something different. As for the emulator, I asked another colleague, and he shared  [https://daintycloud.com/vps-android-emulator/ ](https://daintycloud.com/vps-android-emulator/)  link with me where I managed to review more than one option too.
